### DISCUSSION TITLE
.MOV not playing in browser

### DISCUSSION THREAD

**Discussion by fOO223Fr** [nextcloud/all-in-one#5154] [Questions] [answered]
In firefox it says:
![image](https://github.com/user-attachments/assets/f93a9639-adf7-4252-9969-b8a28d04896c)
and in chrome: only audio plays, no video

**Comment by fOO223Fr** [comment]
funny thing it plays in android memories app just fine

**Comment by szaimen** [comment, marked as answer]
Hi, this is an issue of https://github.com/nextcloud/viewer. Feel free to open an issue there.
